.packit.yaml

```diff
@@ -35,6 +35,11 @@ jobs:
           - https://yum.theforeman.org/plugins/nightly/el9/x86_64/
     module_hotfixes: true
 
+  - job: tests
+    trigger: pull_request
+    targets:
+      - epel-9
```

Comment:
Can this be `rhel-9`? Alternatively we'll probably need to set up additional repos here

Reply:
Can you be more descriptive please I might not get it?

Reply:
The previous build failed with this - apparently it tried to build the rpm without having access to the other repositories we supply for the `copr_build` job.
```
No matching package to install: 'foreman-assets >= 3.15'
No matching package to install: 'foreman-plugin >= 3.15'
No matching package to install: 'npm(@theforeman/builder) >= 15.0.1'
No matching package to install: 'rubygem(deface)'
No matching package to install: 'rubygem(foreman-tasks) >= 8.3.0'
```

With the `rhel-9` thing, I was hoping that if we used the same target for both of those jobs, things would sort themselves out, but apparently that is not an available target on testing farm, so I'm sorry for that. So back to epel-9 and we'll see where it takes us